File: CppCompiler/nestedforloop6.cpp
#include<iostream>
using namespace std;

int main()
{
  int n,i,sum=0,prod=1,j;
cout<<"enter range";
cin>>n;
for(i=1;i<=n;i++)
{
int z=i;
for(j=1;z!=0;j++)
{
sum=sum+(z%10);
prod=prod*(z%10);
z=z/10;
}
if(sum==prod)
{
cout<<i<<endl; 
}
sum=0;
prod=1;
}
return 0;
}
    

File: python programming/nestedforloop6.py
n=int(input("enter no of term:-"))
sum=0
ans=0
for i in range(1,n+1):
	for j in range(1,i+1):
		fact=1
		print(j,end='!')
		for k in range(1,j+1):
			fact=fact*k
		sum=sum+fact
		if j<i:
			print('+',end='')
	print(f'/{i}',end='')
	if i<n:
		print( ' + ',end='')
	ans=ans+sum/i
	sum=0
print('\n sum of seriese=',ans)
	

		
		
